### PR TITLE
[node-polyglot] fixed type for version 2.4.0 (added pluralRules)

### PR DESCRIPTION
**Pull request by dcylabs**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [here](https://github.com/airbnb/polyglot.js/blob/v2.4.0/index.js#L216)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by liamross — APPROVED** [review]
~👍 LGTM~ one change requested

File: types/node-polyglot/index.d.ts [hunk 1]
```diff
@@ -18,13 +18,19 @@ declare namespace Polyglot {
         suffix?: string;
     }
 
+    interface PluralRules {
+        pluralTypes: {[lang: string]: (n: number) => void};
```
Comment (liamross) [inline types/node-polyglot/index.d.ts:22]: Actually, I think potentially this should return a boolean?

**Review by liamross — CHANGES_REQUESTED** [review]
I think the pluralTypes function should return number

**Comment by dcylabs** [comment]
You right ! I'm going to correct these ASAP :)

**Review by liamross — APPROVED** [review]
👍 awesome! looks good

**Pull request merged**